### PR TITLE
chore: version bump 0.2.0

### PR DESCRIPTION
**Pull request by d-jeffery**
## Description

[openfga-vscode-0.2.0.vsix.zip](https://github.com/openfga/vscode-ext/files/12693238/openfga-vscode-0.2.0.vsix.zip)

## Review Checklist
- [ ] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [x] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v0.2.0
+
```
Comment (adriantam) [inline CHANGELOG.md:4]: Usually there is a line showing the diff.  See https://github.com/openfga/js-sdk/blob/cbf282f1666991d2959129292ad6a26369360e63/CHANGELOG.md?plain=1#L5

**Review by adriantam — CHANGES_REQUESTED** [review]
You will need to generate the updated package-lock.json

File: CHANGELOG.md (hunk 1 above)
Comment (d-jeffery) [reply CHANGELOG.md:4]: Added.

Comment (adriantam) [reply CHANGELOG.md:4]: It should be a diff between 0.2.0 and 0.1.0

Comment (d-jeffery) [reply CHANGELOG.md:4]: Done.

**Review by adriantam — APPROVED** [review]

**Pull request merged**